sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/DigitalTwinModelsImpl.java

```diff
@@ -109,7 +110,7 @@
                 @HeaderParam("tracestate") String tracestate,
                 @PathParam("id") String id,
                 @QueryParam("api-version") String apiVersion,
-                @BodyParam("application/json-patch+json") List<Object> updateModel,
+                @BodyParam("application/json-patch+json") JsonPatchDocument updateModel,
```

Comment:
Question - (I know that I am late on these changes) what changed in the swagger file to enable AutoRest to define `JsonPatchDocument` as an acceptable input type?

Reply:
It should just be due to using a newer version of autorest, right?

Reply:
that is an excellent question. code was generated using the latest branch of autorest since these changes were not published to npm yet. There will be a release on autorest and we will bump the autorest version when it is available.

Reply:
Got it!
So this in swagger was automatically mapped to `JsonPatchDocument`?
```json
{
	name: "updateModel",
	description: "An update specification described by JSON Patch. Only the decommissioned property can be replaced.",
	in: "body",
	required: true,
	schema: {
		type: "array",
		items: {
			type: "object"
		}
	}
}
```

How would the schema indicate that this should have been an array of objects vs a `JsonPatchDocument`?

Reply:
I imagine autorest looks for the consumes field instead:
```
        "consumes": [
          "application/json-patch+json"
        ],
```

Reply:
Correct